src/ontology/config/ordo_exclusions.tsv

```diff
@@ -58,58 +58,3 @@ Orphanet:238621	Ileal pouch anal anastomosis related faecal incontinence	TRUE	MO
 Orphanet:505395	Ventilator-induced diaphragmatic dysfunction	TRUE	MONDO:excludePhenotype
 Orphanet:90076	Partial deep dermal and full thickness burns	TRUE	MONDO:excludePhenotype
 Orphanet:90080	Scarring in glaucoma filtration surgical procedures	TRUE	MONDO:excludePhenotype
-MONDO:0850029	antibiotic therapy dose selection	TRUE	MONDO:excludeNonDisease

```

Comment:
@nicolevasilevsky These were added a couple of days ago, but this causes an error in the exclusion expansion script (and slurp, which also uses it as a step), because it only expects terms from the given ontology.

Did you mean to excluded any Orphanet terms that are mapped to these Mondo terms, or some other functionality?  If there is a use case here, we can talk with Nico about how to do, but my guess is that we will want to keep only the given ontology's terms here.

For now, I've removed these rows within this PR.

Reply:
How did THAT happen?

Reply:
No this is just wrong.

Reply:
this is resolved on my end